bin/convert-argv.js

```diff
@@ -94,7 +94,9 @@ module.exports = function(optimist, argv, convertOptions) {
 		configFileLoaded = true;
 	}
 
-	if(typeof options === "function") {
+	var isES6DefaultExportedFunc = (typeof options === "object" && typeof options["default"] === "function");
```

Comment:
Could you add a check for `null`?

``` js
var isES6DefaultExportedFunc = (options && typeof options === "object" && ...
```

Maybe also a few lines below in my code. I've done the same mistake ;)